library/common/jni/jni_interface.cc

```diff
@@ -920,6 +920,11 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_registerEventTracker(JNIEnv* en
                                                                       jobject j_event_tracker) {
   jni_log("[Envoy]", "register event tracker");
 
+  if (j_event_tracker == nullptr) {
```

Comment:
for cases when `j_event_tracker` is `nullptr` we need to register an empty `envoy_event_tracker ` (with its `track` and `context` properties equal to `nullptr`). Otherwise the app will crash when lyne sink attempts to retrieve the registered api in [here](https://github.com/lyft/envoy-mobile-private/blob/master/extensions/stat_sinks/lyne/grpc_metrics_service_impl.cc#L26) and because of [this assertion](https://github.com/envoyproxy/envoy-mobile/blob/main/library/common/api/external.cc#L24)